bot/cogs/bot.py

```diff
@@ -26,7 +26,7 @@ class Bot:
     def __init__(self, bot: AutoShardedBot):
         self.bot = bot
 
-        # Stores allowed channels plus unix timestamp from last call
+        # Stores allowed channels plus unix timestamp from last call.
         self.channel_cooldowns = {HELP1_CHANNEL: 0,
```

Comment:
Could we change the indentation of `self.channel_cooldowns`? There's no need to ignore `E124` if we indent like this:
```py
self.channel_cooldowns = {
    HELP1_CHANNEL: 0,
    HELP2_CHANNEL: 0,
    HELP3_CHANNEL: 0,
    HELP4_CHANNEL: 0,
    PYTHON_CHANNEL: 0,
    DEVTEST_CHANNEL: 0,
    BOT_CHANNEL: 0
}
```